### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_Keyboard: The focus is not visible when the page is opened.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6746]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS ReferencesMAS2.4.3, MAS2.4.7

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select third radio button.
9. Navigate till next button & hit enter.

**Actual:**
On hitting the next button "Select import bacpac settings" page is opened, the focus is not visible on the page, on using tab key the focus is on the "Cancel" button.

**Same issue is observed on the following page:
1.Deploy Dacpac Summary Screen
Follow the above steps and select radio button 1
2.Extract Dacpac Summary Screen
Follow the above steps and select radio button 2
3.Import Bacpac Summary Screen
Follow the above steps and select radio button 3
4.Export Bacpac Summary Screen
Follow the exact same steps and select radio button 4 and click on next button. Again click on the next button.

**Expected:**
On hitting the next button "Select import bacpac settings" page is opened, the focus should be visible on the 1st interactive element that is: edit box beside label "File location" and on tab key the focus should be on the combo-box below it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Users will not understand the flow of navigation on the page.

**MAS Reference:**
MAS2.4.3-[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

MAS2.4.7-[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment for Reference:**
[#6746A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3500723/6746A11y_AzureDataStudio.Schema.Dacpac.Backup._ImportBacpac_ImportBacpacSettings_Keyboard.pptx)

**Comment by udeeshagautam** [comment]
Summary table is not actionable - do we still need focus there?

for rest of pages in dacfx wizard - is this bug dup of https://github.com/microsoft/azuredatastudio/issues/6798

**Comment by fsteffi** [comment]
@udeeshagautam no we don't need the focus on the table for summary screens but it could be on Extract button

**Comment by fsteffi** [comment]
Checked and verified in below version the bug is no longer reproduced and hence closing the bug.
Version: 1.12.0-insider
Commit: a584aca96997a18ecedee73a575536332e542718
Date: 2019-09-19T07:58:50.448Z
VS Code 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

[#6746 A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3662596/6746.A11y_AzureDataStudio.Schema.Dacpac.Backup._ImportBacpac_ImportBacpacSettings_Keyboard.pptx)
